app/views/chapters/_verse.html.erb

```diff
@@ -90,7 +90,7 @@
     <div class="col-md-11">
       <div class="translation">
         <% if @presenter.render_translations? %>
-          <%= render partial: 'translation', collection: verse.translations, as: :translation %>
+          <%= render partial: 'translation', collection: verse.translations.sort_by { |t| @presenter.valid_translations.index(t.resource_content_id) }, as: :translation %>
```

Comment:
I guess a better solution is to fix the data instead. We're soring translation by priority already.  https://github.com/quran/quran.com-frontend-v2/blob/73a66e70882c55d554d0ab89c02babff220684ec/app/presenters/chapter_presenter.rb#L304